pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -53,9 +53,9 @@ in rec {
       # Including it then would cause needless mass rebuilds.
       #
       # TODO(@Ericson2314): Make [ "build" "host" ] always the default.
-      configurePlatforms ? lib.optionals
+      configurePlatforms ? [ "build" ] ++ lib.optional
         (stdenv.hostPlatform != stdenv.buildPlatform)
-        [ "build" "host" ]
+        "host"
```

Comment:
Why leave out host?

Reply:
I think it forced some cross-specific logic, even if both were the same. Don't want to go that far yet :).